requirements.txt

```diff
@@ -1,5 +1,7 @@
-autopep8==1.4.4
-Django==2.2.8
+asgiref==3.2.7
+autopep8==1.5.2
+click==7.1.2
+Django==3.0.6
```

Comment:
2.2 is an LTS version though, so better update to 2.2.12 instead of 3.0.x

Reply:
Good hint. Thanks :)